### ISSUE TITLE
Motivation for vueneue

### ISSUE THREAD

**Issue by abastardi** [vueneue/vueneue#36]
This project seems *very* similar to Nuxt. What is the motivation for creating it, and what differentiates it from Nuxt? Is the main difference that it is based on Vue CLI 3 and compatible with its plugin system? It might be helpful to mention some of this in the readme/documentation intro.

**Comment by camwhite** [comment]
Speaking from my point of view, as a vueneue user, the difference you have already mentioned is a big selling point. Staying inside the vue cli ecosystem has  great benefits in my eyes(yes nuxt 3.0 will have this).

But on top of that, vueneue also allows me to implement my own project structure more in line with traditional vue apps. While I think the nuxt pattern is great for insanely rapid prototypes, I've been warned it can actually add a layer of difficulty as you add more advanced features. 

**tldr;** vueneue is unobtrusive and bends to my development style

**Comment by Tjark-Kuehl** [comment]
I recently switched my project from NuxtJS to vueneue.
Like mentioned the support of the cli ecosystem was a big selling point.
Also the time it takes for NuxtJS to update just takes so long that was a big bummer for me.
And we made a very complex authentication which was really hard to programm because you dont have the full control of what is rendered server side and what is not.
In Vueneue we can use tokens without any problem so its more like server side rally fast renderes views with client side addons.

**Comment by abastardi** [comment]
Thank you. These replies are helpful.

**Comment by abastardi** [comment]
Although this issue has been closed, I still think it would be a good idea to add something to the readme and/or documentation explaining why one might choose vueneue over Nuxt, as that may generate more interest in the project.
